lib/build_runner.zig

```diff
@@ -259,6 +259,10 @@ pub fn main() !void {
                 builder.enable_darling = true;
             } else if (mem.eql(u8, arg, "-fno-darling")) {
                 builder.enable_darling = false;
+            } else if (mem.eql(u8, arg, "-fcompdb")) {
+                builder.enable_compdb = true;
+            } else if (mem.eql(u8, arg, "-fnocompdb")) {
```

Comment:
this should be `-fno-compdb` to match what you added to the help and the other `-fno-` flags

Reply:
@xdBronch Thank you for catching that typo! Fixed.